.env

```diff
@@ -0,0 +1 @@
+COMPOSE_PROJECT_NAME=community
```

Comment:
This requires direnv right?  Or does docker read this natively?

Reply:
It reads it natively: https://docs.docker.com/compose/env-file/